modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -55,12 +55,12 @@ class Const {
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1552584379498
+    static public final long APP_TIMESTAMP = 1552691843205
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 5061
+    static public final int APP_BUILDNUM = 5083
 
 
```

Comment:
Please do not commit changes in Const file, since they are automatically generated.